pycbc/live/snr_optimizer.py

```diff
@@ -177,10 +178,6 @@ def compute_minus_network_snr_pso(v, *argv, **kwargs):
     return -nsnr_array
 
 
-def normalize_initial_point(initial_point, bounds):
-    return (initial_point - bounds[:,0]) / (bounds[:,1] - bounds[:,0])
-
-
 def optimize_di(bounds, cli_args, extra_args, initial_point):
     bounds = numpy.array([
```

Comment:
Suggest to add a comment here saying why we need this - eg `Convert from dict to array with parameters in a given order`